src/integTest/java/com/unosquare/cvgenerator/controller/TemplateControllerIntegrationTest.java

```diff
@@ -0,0 +1,49 @@
+package com.unosquare.cvgenerator.controller;
+
+import com.unosquare.cvgenerator.AbstractWebIntegrationTest;
+import com.unosquare.cvgenerator.dao.TemplateRepository;
+import com.unosquare.cvgenerator.model.entity.TemplateDetail;
+import io.restassured.RestAssured;
+import org.apache.http.HttpStatus;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+
+public class TemplateControllerIntegrationTest extends AbstractWebIntegrationTest {
+
+    @Autowired
+    TemplateRepository repository;
+
+    @Before
+    public void setUp() {
+        TemplateDetail templateDetail1 = new TemplateDetail(1, "testtemplate1", "testtemplatehtml1");
+        TemplateDetail templateDetail2 = new TemplateDetail(2, "testtemplate2", "testtemplatehtml2");
+        repository.save(templateDetail1);
+        repository.save(templateDetail2);
+    }
+
+    @Test
+    public void testGetTemplate() {
```

Comment:
I'd try to go for more meaningful names for your tests. Makes it quicker to resolve problems